code/items/weapons/Shotgun.cs

```diff
@@ -55,13 +59,12 @@ protected override void ShootEffects()
             Particles.Create("particles/pistol_ejectbrass.vpcf", EffectEntity, "ejection_point");
 
             ViewModelEntity?.SetAnimBool("fire", true);
+            CrosshairPanel?.CreateEvent("fire");
```

Comment:
technically isn't needed with the crosshair we currently use.

Reply:
That's right, but kept it for consistency and to be able extending it easily later on.